### ISSUE TITLE
Receiving and distributing a grant

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#823]
Someone recently sent us an email asking if Liberapay could be used by a foundation to send a 6-months grant to a team, in a large one-time payment.

This is currently difficult as our system wasn't designed for that. (There's also the matter of the cumbersome identity verification when giving more than €2500 in a year, but that's a separate issue.)

Here are some ideas on how to improve:

- [ ] Create a `/%username/grant` page in parallel to the existing `/%username/donate` form. The grant form would ask for a payment amount instead of a recurrent donation amount, i.e. the reverse of the normal donation flow.
- [ ] Implement attributing the funds to the specific grant exclusively, so they don't end up funding another grant or a recurrent donation. This could be done in our DB (`cash_bundles`) or by using a separate wallet in Mangopay.
- [ ] Add a `grant` transfer context and modify payday for that.

Under the hood this is technically close to implementing one-off donations (#415), but in the UI it would be different.

Related issues for the distribution side: #753 and #803.
